src/ntimporters/trello/importer.py

```diff
@@ -0,0 +1,344 @@
+"""Trello -> Nozbe Teams importer"""
+import json
+import random
+from typing import Optional, Tuple
+
+import openapi_client as nt
+from dateutil.parser import isoparse
+from ntimporters.trello.trello_api import TrelloClient
+from openapi_client import apis, models
+from openapi_client.exceptions import OpenApiException
+from openapi_client.model.id16_read_only import Id16ReadOnly
+from openapi_client.model.id16_read_only_nullable import Id16ReadOnlyNullable
+from openapi_client.model.timestamp_read_only import TimestampReadOnly
+from openapi_client.model_utils import ModelNormal
+
+
+class ImportException(Exception):
+    """Import exception"""
+
+
+def strip_readonly(model: ModelNormal):
+    """Strip read only fields before sending to server"""
+    for field in [
+        elt
+        for elt in model.attribute_map.values()
+        if hasattr(model, elt)
+        and isinstance(getattr(model, elt), (Id16ReadOnly, Id16ReadOnlyNullable, TimestampReadOnly))
+    ]:
+        del model.__dict__.get("_data_store")[field]
+    return model
+
+
+def id16():
+    """Generate random string"""
+    return 16 * "a"
+
+
+SPEC = {
+    "code": "trello",  # codename / ID of importer
+    "name": "Trello",  # name of application
+    "url": "https://developer.atlassian.com/cloud/trello/guides/rest-api/api-introduction/",
+    "input_fields": ("nt_auth_token", "auth_token", "app_key"),
+}
+COLORS = [
+    "aquamarine",
+    "aubergine",
+    "blue",
+    "brown",
+    "burntsienna",
+    "darkgreen",
+    "deeppurple",
+    "dustpink",
+    "green",
+    "heather",
+    "indigo",
+    "karmin",
+    "lightblue",
+    "lightpink",
+    "mauve",
+    "midnight",
+    "navy",
+    "ocean",
+    "ocher",
+    "olive",
+    "orange",
+    "pink",
+    "purple",
+    "red",
+    "sand",
+    "stone",
+    "taupe",
+    "teal",
+    "ultramarine",
+]
+
+
+# main method called by Nozbe Teams app
+def run_import(nt_auth_token: str, auth_token: str, app_key: str, team_id: str) -> Optional[str]:
```

Comment:
there should be a PR in Nozbe Teams repo to enable calling `run_import` with different number of aruments than default 3 (`nt_auth_token`, `auth_token`, `team_id`)

Reply:
i've created and assigned you there